src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -0,0 +1,195 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="quick-navigation-modal">
+    <div class="container">
+      <div
+        class="close-icon"
+        @click="closeQuickNavigationModal()"
+      >
+        <p>
+          Close
+        </p>
+      </div>
+      <input
+        ref="input"
+        class="filter"
+        v-model="userInput"
+        type="text"
+      />
+      <div>
+        <div
+          v-for="(symbol, idx) in filteredSymbols"
+          class="symbol-match"
+          @click="closeQuickNavigationModal()"
+          :key="idx"
+        >
+          <Reference :url="symbol.path" :id="idx">
+              -
+              {{ symbol.title.slice(0, symbol.start) }}
+              <QuickNavigationHighlighter
+                :text="symbol.substring"
+                :matcherText="debouncedInput"
+              />
+              {{ symbol.title.slice(symbol.start + symbol.matchLength) }}
```

Comment:
Maybe adding this text into a `span` tag?

Reply:
Added in https://github.com/apple/swift-docc-render/pull/372/commits/c5658e5da7a0139286f8ec6eb50ce4fb9577b20a